radixdlt/src/integration/java/com/radixdlt/consensus/simulation/SimulatedBFTNetwork.java

```diff
@@ -176,8 +180,24 @@ public SystemCounters getCounters(ECKeyPair keyPair) {
 		return counters.get(keyPair);
 	}
 
-	public void start() {
+	public Completable start() {
+		// Send start event once all nodes have reached real epoch event (first epoch event is empty)
+		final CompletableSubject completableSubject = CompletableSubject.create();
+		List<Completable> startedList = this.runners.values().stream()
+			.map(ConsensusRunner::events)
+			.map(o -> Completable.fromSingle(
+				o.map(Event::getEventType)
+					.filter(e -> e.equals(EventType.EPOCH))
+					.skip(1)
+					.firstOrError()
+				)
```

Comment:
Might be more idiomatic to express this as `o.map(..)...firstOrError().ignoreElement()` to make it clear that we're intentionally dropping the value here

Reply:
ah! i was looking for `toCompletable`, but `ignoreElement` is what I want :)